personio-personnel-data-api-oa3.yaml

```diff
@@ -1123,7 +1123,7 @@ paths:
         - name: offset
           in: query
           required: false
-          description: Pagination attribute to identify which page you are requesting, by the form of telling an offset from the first record that would be returned.
+          description: The offset from the first record that would be returned. With 3 results and an offset of 1, 2 results are returned.
```

Comment:
maybe could be useful to provide an example of the order results are returned and which items will be returned with the offset . 

`For example with 3 results [ ResultA, ResultB, ResultC ] and an offset of 1, the following results will be returned: [ ResultB, ResultC ] returned.`